framework/src/modules/chain/blocks/blocks.js

```diff
@@ -846,6 +845,27 @@ class Blocks extends EventEmitter {
 			}`
 		);
 	}
+
+	/**
+	 * Handle the fast switching mechanism
+	 *
+	 * @param lastBlock
+	 * @param block
+	 * @return {Promise<void>}
+	 * @private
+	 */
+	// eslint-disable-next-line no-unused-vars, class-methods-use-this, no-empty-function
+	async _handleFastChainSwitching(lastBlock, block) {}
```

Comment:
what do we mean by handle here?

Reply:
`_handle` follows the same naming consistency across all logic that gets triggered as a result of the fork choice rules

Reply:
Isn't the handling part going to be implemented in each synchronization mechanism?

Reply:
Agreed, I thought this was old code but it is indeed introduced here. @nazarhussain could you elabore more on why we need this here?